lib/sdk-installer.js

```diff
@@ -53,6 +53,9 @@ function installAndroidSdk(apiLevel, target, arch, channelId, emulatorBuild, ndk
             const isOnMac = process.platform === 'darwin';
             const isArm = process.arch === 'arm64';
             const cmdlineToolsPath = `${process.env.ANDROID_HOME}/cmdline-tools`;
+            if (fs.existsSync(cmdlineToolsPath)) {
+                yield io.rmRF(cmdlineToolsPath);
+            }
```

Comment:
This may be a simpler / better approach than #411 - but with 411 there is the possibility we skip the install if the runner image ever has the updated version. That may have value

However I see this nice `io.rmRF` call and that's better than I do in 411 - I was shelling out to rm things